src/app/_components/Feedback/Feedback.js

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import LoveIcon from '@icons/loveIcon.svg';
+import { useCreateFeedback } from '@/app/_hooks';
+import SendFeedback from '@/lib/validationSchemas/sendFeedbackSchema';
+import { Modal } from '../Modal';
+import { TextInputField } from '../InputFields';
+import { CheckBox } from '../CheckBox';
+import { TextArea } from '../TextArea';
+import { PillButton } from '../PillButton';
+import { cn } from '@/utils/cn';
+import { buttonColorVariant, buttonVariant } from '../PillButton/style';
+
+const MAX_LENGTH = 320;
+
+export function Feedback({ isFeedbackOpen, onClose }) {
+  const [isFormOpen, setFormOpen] = useState(true);
```

Comment:
🟢 maybe better to do is otherwise - `isSubmitted` ?

Reply:
IF **isFormOpen === true**   _I show Feedback form_

IF **isFormOpen === false** _I hide form and show "Thank You Modal"_